### PR TITLE
[Delivers #96974108] skip Tier 1 for WHSC

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

File: app/models/ncr/work_order.rb [hunk 1]
```diff
@@ -114,7 +115,35 @@ def name
       self.project_title
     end
 
+    def system_approvers
+      results = []
+      if self.expense_type == 'BA61'
+        unless self.organization.whsc?
+          results << self.class.ba61_tier1_budget_mailbox
+        end
```
Comment (afeld) [inline app/models/ncr/work_order.rb]: This is the important part...the rest is simply refactoring to support it.

File: app/models/ncr/work_order.rb [hunk 2]
```diff
@@ -97,6 +92,12 @@ def client
       "ncr"
     end
 
+    def organization
+      # TODO reference by `code` rather than storing the whole thing
+      code = self.org_code.split(' ', 2)[0]
```
Comment (cmc333333) [inline app/models/ncr/work_order.rb]: Is it worth accounting for a nil `org_code` here?

File: lib/ncr/organization.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    end
+  end
+end
+
+rows = CSV.read("#{Rails.root}/config/data/ncr/org_codes_2015-05-18.csv", headers: true)
+
+Ncr::Organization::BY_CODE = {}
+rows.each do |row|
+  org = Ncr::Organization.new(row)
+  Ncr::Organization::BY_CODE[org.code] = org
```
Comment (cmc333333) [inline lib/ncr/organization.rb]: This is a solution to our duplicate org code problem, too

**Comment by afeld** [comment]
Updated!

File: app/models/ncr/work_order.rb [hunk 4]
```diff
@@ -114,7 +115,35 @@ def name
       self.project_title
     end
 
+    def system_approvers
+      results = []
+      if self.expense_type == 'BA61'
+        unless self.organization.whsc?
```
Comment (cmc333333) [inline app/models/ncr/work_order.rb]: To account for nils, maybe a `try` here?

Comment (afeld) [inline app/models/ncr/work_order.rb]: Good catch!

**Comment by afeld** [comment]
Updated!

**Comment by cmc333333** [comment]
Thanks!

**Pull request merged**